README.md

```diff
@@ -57,6 +57,27 @@ In Twig!
 {{ time_diff(someDateTimeVariable) }}
 ```
 
+In Symfony controllers
+
+Let's suppose you want to return a custom response like JSON, 
+you can use the DateTimeFormatter service in your controllers.
+```
+public function yourAction()
+{
+    ...
+    $dateTimeFormatter = $this->get('time.datetime_formatter');
```

Comment:
We really need to show the autowiring way of getting this service that’s used in Symfony 4. We should also show the $this->get(), but commented out. 

Do you know about the autowiring way? Let me know :)